applications/madini_tracechain.md

```diff
@@ -134,7 +151,7 @@ www.madinitracechain.com
 
 
 Madini MDN Token
-
+============================================================
 MDN VIP180 is the Madini utility token in our ecosystem
```

Comment:
The MDN token is the foundation of Madini ecosystem that will enable holders to access several features and TraceChain. It is meant to serve as a multi-purpose token rewarding early supporters of the project, users and stakeholders and create true value for our ecosystem.

MDN token is generated at the creation of a TraceChain user account, and on eachTraceability record (products upload), and will start generating MDN tokens.

Madini (MDN) VIP180 token

MDN smart contract is build as VIP180 VeChain token. With full usability in Madini Ecosystem, anyone can earn, trade, transfer, pay, receive and invest in Madini MDN. MDN utility token main purpose is paying the services of traceability and is used for the purchases/selling of the minerals/metals.

As every stakeholder of the marketplace will have a VeChain wallet, MDN will be used as Trading fee and also as payment option for the minerals.

We have created a VeChain Thor – Madini Wallet (Android) to facilitate the Token management and allow the general public to invest, transfer and trade the MDN/VET/VTHO tokens.

MDN – Main token MDN – Main value: adoption and utility (MDN is linked to all the Madini products, main CC for the Madini Wallet and the Madini Marketplace)

Initial Value 0,01 USD (Aprox 1 KSH) Total supply: 100,000,000 MDN Decimal positions: 18
[Status: LIVE on VeChain Thor MainNet 5th Jan 2021](https://explore.vechain.org/accounts/0x1b44a9718e12031530604137f854160759677192)

![MDN in a liquidity pool on VeExchange (courtesy of MDN user)](https://user-images.githubusercontent.com/8124128/140723636-6a0c11ac-4ec8-49cc-b3f6-5d1304c6cd94.jpg)




MDN Vip180 is available on all VeChain ecosystem. You can use the new SYNC2 VeChain wallet or Vexchange to swap it. Start earning providing liquidity pools with MDN on Vexchange. # VeChain Sync2 Sync etc

Sync2 is designed to work with all mainstream web browsers (e.g., Chrome, Safari, MS Edge, Firefox, etc), desktop, and mobile phones. Use it to easily interact with Vexchange (image above) from yur phone or computer.

Using VeChain Sync

Sync is the official desktop wallet from Vdechain. It provides seamless experience for users. Madini MDN is part of VeChain blockchain as VIP180 token so any VeChain wallet can operate with it. It is not neccesary to create a new specific wallet. When you purchase MDN, MyTokens will automatically reflect the balance on Sync .

More information: [https://www.madinitracechain.com/mdn-vechain-vip180/](https://www.madinitracechain.com/mdn-vechain-vip180/)